### ISSUE TITLE
Change Request: Add notice to user of potential delay generating L1 Keys

### ISSUE THREAD

**Issue by rabsef-bicrym** [urbit/bridge#1033]
**Describe the bug**
I burned two rifts thinking that maybe I had somehow corrupted a keyfile on a ship I was breaching as the urbit runtime was telling me the key didn't match the public-key state. Waiting some 10 minutes allowed the third keyset I'd generated to be valid and boot the planet. This is an L1 planet - I believe that the delay may be accounted for by routing the traffic thru the rollup rather than directly to a node, but I am not sure. It, in any event, is a departure from prior behavior.

**To Reproduce**
1. Breach a ship (L1)
2. Immediately attempt to boot from the generated keyfile

**Expected behavior**
Either notify the user that they will need to wait or do not provide the keyfile until the network change has saturated and one can boot from it.

**Comment by matildepark** [comment]
Yes, I hit this too all the time — I have to tell people if they breach that it'll be about a half hour and it'll keep telling them they have the wrong keys but it's just that Azimuth now takes like a half hour to see what they did

**Comment by jeremytunnell** [comment]
IMO, using a timer like we do with invites is a better solution than yet more warnings and instructions.

**Comment by thelifeandtimes** [comment]
this is in the `dawn.c` file in vere. line 278 in the `u3_dawn_vent()` function (validated boot event). I looked at this with @matthew-levan and it is looking at roller.urbit.org/v1/azimuth which presumably is actually a roller instead of directly looking at L1 state as it comes in from ethereum. Interestingly enough, this means that point #2 of the comment above is actually relevant.

This constraint means the delay is, at minimum, linked to the existing block finality protections of `%eth-watcher` which gives us something on the order of a 5 minute delay, instead of, say, the 15 second block time for ethereum. 

The quick fix here remains just clearer messaging in bridge and related guides that patience is a virtue, but ideally this is fixed in Vere so that L1 points can breach and reboot quickly.
